frida_tools/repl.py

```diff
@@ -335,6 +340,8 @@ def _process_input(self, reactor):
                         if expression.startswith("%"):
                             self._do_magic(expression[1:].rstrip())
                         else:
+                            if self._autoperform:
+                                expression = f"Java.performNow(() => {{return {expression}\n/**/}});"
```

Comment:
```suggestion
                                expression = f"Java.performNow(() => {expression});"
```

Could this be simplified to this perhaps? (Untested)

Reply:
Since I'm not actually parsing the code but just concatenating strings I tried to use the most versatile expression I could think of.

* The double curly braces is escaping of a single curly brace because this is inside a Python 3 string format. But now that I think about it, this is a problem because we still support Python 2, I will fix it.
* The line break is used so we won't fail if the inserted code ends with a line comment
* The `/**/` section is used so we won't fail if the inserted code ends with an open multi line comment